GitUI/UserEnvironmentInformation.cs

```diff
@@ -9,18 +9,13 @@ namespace GitUI
     public sealed class UserEnvironmentInformation
     {
         private static bool _alreadySet;
-        private static bool _dirty;
-        private static string _sha;
+        private static bool _dirty = true;
```

Comment:
Gee. Don't you think I might want to use valid data so anybody designing the form would see the length of the string and see the clipping if they resized the form. 
•••

Reply:
I actually do put some thought and why I do things. It sure seems like you doubt it. Yes. I do miss things sometimes. I do appreciate feedback but sometimes I think you and others don't believe it and act like we are idiots.

Reply:
For Designer specific, use specific tests:
https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.licensemanager.usagemode?redirectedfrom=MSDN&view=netframework-4.7.2#System_ComponentModel_LicenseManager_UsageMode

If you see a value like this, you wonder what the significance of the default value is. 
It is just hard to review.

Reply:
Reasons why I used those values
- Easy value to say hmmm  why am I seeing this value in runtime
- Includes Dirty state to include dirty status
- Includes characters and digits for font kerning issues.

Reply:
I chose not to do design checking because the control is loading the data from this source.  The source doesn't know about design time.  The source could be used in other places that don't know about design time.

Reply:
> Gee. Don't you think I might want to use valid data so anybody designing the form would see the length of the string and see the clipping if they resized the form.
>
> I actually do put some thought into why I do things. It sure seems like you doubt it. Yes. I do miss things sometimes. I do appreciate feedback but sometimes I think you and others don't believe it and act like we are idiots.

I think @gerhardol's question was reasonable. I see no indication that anyone is treating you disrespectfully Jay, and I feel others deserve that same respect from you. We are all here because we have chosen to be, so let's please keep it a nice place to hang out. Personally, I try to assume good intentions in others and am generally happier for it.

Reply:
I also didn't want a chance of changes to design checking to change the expected value. I expect the source to control the data always. If I did design checking we would have to maintain two sources. 

@drewnoakes sure. I get it. I just felt like he thought I  was just doing things half way or not thought out. Sometimes my implementation misses but I do try to come up with solutions that cover all the bases. 

@gerhardol. Sorry. I just felt like all my thought/work was just being treated as I just did some simple thing and wasn't thought out.